### ISSUE TITLE
Explore ways to remove non-applicable properties in manure treatment config objects

### ISSUE THREAD

**Issue by ltp23** [RuminantFarmSystems/RuFaS#1303]
Currently, the `ManureTreatmentConfig` class contains a catch-all collection of properties that are used in any of the available manure treatment options. The logical problem with this approach is that it doesn't make sense to have properties such as `storage_time_period` or `composting_type` if the selected manure treatment doesn't use these properties.

I think this issue is of fairly low priority and the solution may disrupt the existing structure quite a bit.

**Comment by ltp23** [comment]
For those properties that are present in some but not all manure treatments, maybe we can add an extra option "N/A" so that all the treatments have the same set of properties.

**Comment by Isgotkowitz** [comment]
@isgotkowitz mention this in replacement issue of #1118 and make sure solution is called for in manure requirements doc.

**Comment by Isgotkowitz** [comment]
Blocked by #1274 - introducing nullability into the metadata. After this issue, explore if nullability would be an appropriate way to address this issue.

**Comment by ew3361zh** [comment]
Unblocking this as #1274 is now completed.

**Comment by matthew7838** [comment]
With new manure design, closed as not planned.
